packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/diagramfilter/services/DiagramFilterDescriptionProvider.java

```diff
@@ -0,0 +1,333 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.views.diagramfilter.services;
```

Comment:
None of the work being done here is related to the views of the workbench. All your code in Sirius Web Application should be in `sirius-web-application/src/main/java/org.eclipse.sirius.web.application.diagram.services.filter`

Reply:
Done (and moved the interfaces in `diagram.services.filter.api`).
For the code in `sirius-web-services` I kept the package `services.diagramfilter`, mimicking `services.relatedelements`.